omnigibson/envs/env_base.py

```diff
@@ -186,7 +186,7 @@ def _load_scene(self):
 
         # Set the simulator settings
         # NOTE: This must be done BEFORE the scene is loaded, or else all vision sensors can't retrieve observations
-        og.sim.set_simulation_dt(physics_dt=self.physics_timestep, rendering_dt=self.action_timestep)
+        og.sim.set_simulation_dt(physics_dt=(1 / self.physics_frequency), rendering_dt=(1 / self.action_frequency))
```

Comment:
should we in general do `1.0 / frequency` to be 100% safe? I know probably no one will use this old a version of python...